packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -65,29 +75,44 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
   }
 
   public render(): React.ReactElement<{}> {
-    const { ariaLabel, className, disabled, label, max, min, showValue, buttonProps, vertical } = this.props;
+    const {
+      ariaLabel,
+      className,
+      disabled,
+      label,
+      max,
+      min,
+      showValue,
+      buttonProps,
+      vertical,
+      styles,
+      theme
+    } = this.props;
     const { value, renderedValue } = this.state;
     const thumbOffsetPercent: number = ((renderedValue! - min!) / (max! - min!)) * 100;
     const lengthString = vertical ? 'height' : 'width';
     const onMouseDownProp: {} = disabled ? {} : { onMouseDown: this._onMouseDownOrTouchStart };
     const onTouchStartProp: {} = disabled ? {} : { onTouchStart: this._onMouseDownOrTouchStart };
     const onKeyDownProp: {} = disabled ? {} : { onKeyDown: this._onKeyDown };
+    const classNames = getClassNames(styles, {
+      className,
+      rootIsDisabled: disabled,
+      rootIsEnabled: !disabled,
+      rootIsVertical: vertical,
+      rootIsHorizontal: !vertical,
```

Comment:
I know it was in original code, but isn't it enough to just have one of each of these variables in the styles props interface? Seems redundant...

Reply:
um very much agreed. I'm gonna clean this up